src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_scenarios_non_ascii_chars.py

```diff
@@ -0,0 +1,64 @@
+                             src_container, source_file, blob_name)
+            self.storage_cmd('storage blob exists -c {} -n {}', storage_account_info,
+                             src_container, blob_name).assert_with_checks(JMESPathCheck('exists', True))
+            url = self.storage_cmd('storage blob url -n {} -c {}', storage_account_info,
+                                   blob_name, src_container).get_output_in_json()
+            self.storage_cmd('storage blob copy start -c {} -b {} -u {}',
+                             storage_account_info, dst_container, blob_name, url).assert_with_checks([
+                                 JMESPathCheck('status', "success")])
+            self.storage_cmd('storage blob exists -c {} -n {}', storage_account_info,
+                             dst_container, blob_name).assert_with_checks(JMESPathCheck('exists', True))
+
+        self.storage_cmd('storage blob copy start-batch --destination-container {} --pattern * --source-container {}',
+                         storage_account_info, dst_container, src_container).assert_with_checks(
+                             JMESPathCheck('length(@)', 2))
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_storage_file_copy_non_ascii(self, storage_account_info):
+        src_share = self.create_share(storage_account_info)
+        dst_share = self.create_share(storage_account_info)
+
+        source_dir = self.create_temp_dir()
+        files = ['fileğşŞ', 'file的Φ']
+        for file_name in files:
+            with open(os.path.join(source_dir, file_name), 'wb') as temp_file:
+                temp_file.write(bytearray([0] * 1024))
+            self.storage_cmd('storage file upload -s {} --source "{}"', storage_account_info,
+                             src_share, os.path.join(source_dir, file_name))
+            self.storage_cmd('storage file exists -s {} -p "{}"', storage_account_info,
+                             src_share, file_name).assert_with_checks(JMESPathCheck('exists', True))
+            url = self.storage_cmd('storage file url -p "{}" -s {}', storage_account_info,
+                                   file_name, src_share).get_output_in_json()
+            self.storage_cmd('storage file copy start -s {} -p "{}" -u {}',
+                             storage_account_info, dst_share, file_name, url).assert_with_checks([
+                                 JMESPathCheck('status', "success")])
+            self.storage_cmd('storage file exists -s {} -p "{}"', storage_account_info,
+                             dst_share, file_name).assert_with_checks(JMESPathCheck('exists', True))
+
+        self.storage_cmd('storage file copy start-batch --destination-share {} --pattern * --source-share {}',
+                         storage_account_info, dst_share, src_share).assert_with_checks(
+                             JMESPathCheck('length(@)', 2))
```

Comment:
Where are the recordings for the tests?

Reply:
I made the tests LiveScenarioTests, so no recordings. 
This was due to me encountering encoding errors when logging messages during playback. 
I didn't want to go about investigating and changing the testbase, hence live-only.
Possible future work.

Reply:
@troydai If you're okay with this only being a live test, I am okay.

Reply:
Let's make it live only for now.